core/templates/utility/string-utility.ts

```diff
@@ -16,11 +16,37 @@
  * @fileoverview Utility functions for strings.
  */
 
-export default function camelCaseFromHyphen(str: string): string {
+export function camelCaseFromHyphen(str: string): string {
```

Comment:
Why this change?

Reply:
By default, only one function existed in the utility file. Since I added a new one, removed the default export part.